TACC/Core-CMS-Custom
# hotfix: outdated purple accent, pages w/ c-message

## Overview

Override outdated colors that are overriding Core-CMS fix for those outdated colors.

## Related

Uncertain.

## Changes

- **added** stylesheet to override outdated colors

## Testing

1. Open https://localhost:8000/register/request-to-submit/.
2. Verify accent color is **blue** (not purple).

## UI

| before | after |
| - | - |
| ![before](https://github.com/TACC/Core-CMS-Custom/assets/62723358/4b3f055a-b519-4984-bed1-25bca9420b69) | ![after](https://github.com/TACC/Core-CMS-Custom/assets/62723358/84c2146c-0c09-4bc9-b662-ad80b58dace8) |

## Notes

<details><summary><strong>Wait, what's happening? Who was overriding who's what?</strong></summary>

ℹ️ Register page loads `c-message.css` directly from Core-Styles at v0.13.0.
✅ Home page loads colors from Core-CMS `site.css` which has new/fixed colors.
❌ That old version of `c-message.css` bundles colors within it, thus pages use old colors.
⚠️ The `c-message.css` colors were overriding Core-CMS `site.css`.

| page **sans** error | page **with** error |
| - | - |
| ![home page](https://github.com/TACC/Core-CMS-Custom/assets/62723358/72ddab80-d8e5-4df5-b8d6-cec1f9d4c06c) | ![register page](https://github.com/TACC/Core-CMS-Custom/assets/62723358/4b3cbd34-9ae8-4a3c-b64a-fcc9ac9eb5fd) |

_So…_

💡 I loaded Core-Styles v2 `color.css` to override `c-message.css`.

_I could have changed to load `c-message.css` from Core-Styles v2, but then I need to test messages. I'm fixing the specific problem — colors — quickly instead._

📝 The real fix is updating APCD-CMS to Core-Styles v2 via setting, and testing.

</details>
